test/fail_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+NC='\033[0m'
+CYAN='\033[0;36m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+
+INPUT_FILES="compiler/fail/*.ods"
+TMP_FILE="temp.py"
+printf "${CYAN}Running compiler tests...\n${NC}"
+
+for input_file in $INPUT_FILES; do
+    expected_error_file=${input_file/.ods/.out}
+    exp_err=$(tr -d "[:space:]" < $expected_error_file)
```

Comment:
Do we want to get rid of the spaces in the error messages? Doesn't really matter too much, but might make more sense to do a straight `cmp` as we do when checking the compiled python file against the expected one (since the spaces are part of the error and not just for readability)